src/bulkExport/startExportJob.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Handler } from 'aws-lambda';
+import AWS from 'aws-sdk';
+import { BulkExportStateMachineGlobalParameters } from './types';
+
+export const startExportJobHandler: Handler<
+    BulkExportStateMachineGlobalParameters,
+    BulkExportStateMachineGlobalParameters
+> = async event => {
+    const { GLUE_JOB_NAME } = process.env;
+    if (GLUE_JOB_NAME === undefined) {
+        throw new Error('GLUE_JOB_NAME environment variable is not defined');
+    }
+    const glue = new AWS.Glue();
+    const startJobRunResponse = await glue.startJobRun({ JobName: GLUE_JOB_NAME }).promise();
```

Comment:
Can we provide a `jobId` to `startJobRun`? This can be used by the job script to place the exported files in the correct bucket.
For example all exported files for job `abcd1234` would go into bucket 
`export/abcd1234`